app/components/primer/alpha/action_list/form_wrapper.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+module Primer
+  module Alpha
+    class ActionList
+      # Utility component for wrapping ActionLists or individual ActionList::Items in forms.
+      class FormWrapper < Primer::Component
```

Comment:
This component seems general enough to make me wonder, does it need to be a subcomponent of `ActionList`?

Reply:
Hmm that's a good question. We could always extract it when we have a secondary use-case for it.